src/components/widgets/ClockWidget.tsx

```diff
@@ -0,0 +1,132 @@
+import styled from 'styled-components'
+import { useRef } from 'react'
+
+const Clock = ({ className }) => {
+  setInterval(setClock, 1000)
```

Comment:
Please change this to `useInterval`, otherwise multiple interval handlers will be added on dashboard switch.